src/views/Home.vue

```diff
@@ -23,9 +23,13 @@ export default {
   },
   methods: {
     pollData() {
-      this.$store.dispatch('getBalance')
-      this.$store.dispatch('getTransactions', { limit: 50, page: 0 })
-      this.$store.dispatch('getAddresses')
+      const currentRoute = this.$router.currentRoute.path
+      const matchRoute = currentRoute.includes('/welcome-back' || '/ftu')
```

Comment:
I had to look it up, `'/welcome-back' || '/ftu'` evaluates to `'/welcome-back'`, thanks javascript.
```suggestion
      const matchRoute = currentRoute.includes('/welcome-back')  || currentRoute.includes('/ftu')
```
Also, shouldn't this use `.startsWith()` or similar?